webpack/ForemanInventoryUpload/Components/PageHeader/components/CloudConnectorButton/CloudConnectorSelectors.js

```diff
@@ -1,14 +1,20 @@
-import { selectAPIResponse } from 'foremanReact/redux/API/APISelectors';
+import { foremanUrl } from '../../../../../ForemanRhCloudHelpers';
+import { selectAccountsList } from '../../../AccountList/AccountListSelectors';
+import { CONNECTOR_STATUS } from './CloudConnectorConstants';
 
-import {
-  CONFIGURE_CLOUD_CONNECTOR,
-  CONNECTOR_STATUS,
-} from './CloudConnectorConstants';
+export const SelectCloudConnectorStatus = state =>
+  selectAccountsList(state).CloudConnectorStatus || {};
 
-export const selectStatus = state =>
-  selectAPIResponse(state, CONFIGURE_CLOUD_CONNECTOR)?.result === 'success'
-    ? CONNECTOR_STATUS.RESOLVED
-    : CONNECTOR_STATUS.NOT_RESOLVED;
+export const selectStatus = state => {
+  const { task } = SelectCloudConnectorStatus(state);
+  if (!task) return CONNECTOR_STATUS.NOT_RESOLVED;
+  if (task.result === 'running' || task.result === 'pending')
+    return CONNECTOR_STATUS.PENDING;
+  if (task.result === 'success') return CONNECTOR_STATUS.RESOLVED;
+  return CONNECTOR_STATUS.NOT_RESOLVED;
+};
```

Comment:
```suggestion
// in Settings.js
const CONNECTOR_STATUS_MAP = {
  running: CONNECTOR_STATUS.PENDING,
  pending: CONNECTOR_STATUS.PENDING,
  success: CONNECTOR_STATUS.RESOLVED,
};

// in selectors.js
export const selectStatus = state => {
  const { task } = SelectCloudConnectorStatus(state);
  return task.result ? CONNECTOR_STATUS_MAP[task.result] : CONNECTOR_STATUS.NOT_RESOLVED;
};
```

Reply:
I made a mistake here there is not "running" result, only "pending", I think in this case using this suggestion edit might complicate it